### ISSUE TITLE
translated man pages are not picked up when building in parallel build

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1982]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1978
- Created at 2013-06-11 16:40:55 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:18:31 as wontfix
- Assigned to nobody
---

Me and Stephen noticed this bug when working on ticket #1510. It seems that the translated man pages are only correctly generated when building in the same directory where the sources are present. Parallel builds don't work correctly.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-06-13 16:13:47](https://pagure.io/SSSD/sssd/issue/1978#233724)**

Minor priority -- as far as I know the bug has always been there and there is an easy workaround. Moreover, normally only the upstream performs releases where the translations are prepared, so this is not affecting users in general.

milestone: NEEDS_TRIAGE => SSSD 1.12
priority: major => minor

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-06-13 16:20:33](https://pagure.io/SSSD/sssd/issue/1978#233725)**

internal build system issue. No clone.

rhbz:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-12 20:52:56](https://pagure.io/SSSD/sssd/issue/1978#233726)**

Fields changed

mark:  => 0
milestone: SSSD 1.13 => SSSD 1.14 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-02-16 14:14:49](https://pagure.io/SSSD/sssd/issue/1978#233727)**

This ticket was opened 3 years ago and so far the maintainers were able to cope up with this defect..patches welcome.

milestone: SSSD 1.14 beta => SSSD Deferred
sensitive:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:48:13](https://pagure.io/SSSD/sssd/issue/1978#233728)**

**Metadata Update from @jhrozek**:
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:18:30](https://pagure.io/SSSD/sssd/issue/1978#636898)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:18:32](https://pagure.io/SSSD/sssd/issue/1978#636899)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
